storage-plans.yaml

```
@@ -1,52 +1,188 @@
-    volume-size: 10
+    volumesize: 240
     instances: 1
     master: false
-    instance: c6g.large.search
+    instance: t2.medium.search
 
+  # 3 dedicated VMs, 1 CPU per VM, 4GB RAM per VM, 240GB disk space
   small-ha:
-    volume-size: 10
-    instances: 2
+    volume-size: 240
+    instances: 3
     master: false
-    instance: c6g.large.search
+    instance: t2.medium.search
 
+  # 3 dedicated VMs, 2 CPU per VM, 8GB RAM per VM, 525GB disk space
+  medium:
+    volumesize: 525
+    instances: 1
+    master: false
+    instance: m6g.large.search
+
+  # 3 dedicated VMs, 2 CPU per VM, 8GB RAM per VM, 525GB disk space
   medium-ha:
-    volume-size: 10
-    instances: 2
+    volumesize: 525
+    instances: 3
+    master: false
+    instance: m6g.large.search
+
+  # 2 CPU per VM, 15GB RAM per VM, 1050GB disk space
+  large:
+    volumesize: 1050
+    instances: 1
     master: false
-    instance: c6g.large.search
+    instance: m6g.xlarge.search
 
+  # 3 dedicated VMs, 2 CPU per VM, 15GB RAM per VM, 1050GB disk space
   large-ha:
-    volume-size: 10
-    instances: 2
+    volumesize: 1050
+    instances: 3
     master: false
-    instance: c6g.large.search
+    instance: m6g.xlarge.search
+
+  # 4 CPU per VM, 31GB RAM per VM, 2100GB disk space
+  x-large:
+    volumesize: 2100
+    instances: 1
+    master: false
+    instance: m6g.2xlarge.search
+
+  # 3 dedicated VMs, 4 CPU per VM, 31GB RAM per VM, 2100GB disk space
+  x-large-ha:
+    volumesize: 2100
+    instances: 3
+    master: false
+    instance: m6g.2xlarge.search    
+
+# RDS/Postgres Instances
+postgres:
+  # 5GB Storage, Postgres Version 13, DB Instance Class: db.t3.micro
+  tiny-13:
+    volumesize: 5
+    replicas: 0
+    instance: db.t3.micro
+
+  # 100GB Storage, Postgres Version 13. DB Instance Class: db.t3.small
+  small-13:
+    volumesize: 100
+    replicas: 0
+    instance: db.t3.small
+  
+  # 100GB Storage, Postgres Version 13. DB Instance Class: db.t3.small
+  small-13-ha:
+    volumesize: 100
+    replicas: 2
+    instance: db.t3.small
+
+  # 100GB Storage, Postgres Version 13. DB Instance Class: db.m5.large
+  medium-13:
+    volumesize: 100
+    replicas: 0
+    instance: db.m5.large
+
+  # 100GB Storage, Postgres Version 13. DB Instance Class: db.m5.large
+  medium-13-ha:
+    volumesize: 100
+    replicas: 2
+    instance: db.m5.large
+
+  # 564GB Storage, Postgres Version 13. DB Instance Class: db.m5.2xlarge
+  large-13:
+    volumesize: 564
+    replicas: 0
+    instance: db.m5.2xlarge
+  
+  # 564GB Storage, Postgres Version 13. DB Instance Class: db.m5.large
+  large-13-ha:
+    volumesize: 564
+    replicas: 2
+    instance: db.m5.2xlarge
+
+  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
+  xlarge-13:
+    volumesize: 2100
+    replicas: 0
+    instance: db.m5.4xlarge
+  
+  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
+  xlarge-13-ha:
+    volumesize: 2100
+    replicas: 2
+    instance: db.m5.4xlarge
```

I could be wrong here, but was the `2100` intentional?

```suggestion
  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
  xlarge-13:
    volumesize: 2000
    replicas: 0
    instance: db.m5.4xlarge
  
  # 2TB Storage, Postgres Version 13. DB Instance Class: db.m5.4xlarge
  xlarge-13-ha:
    volumesize: 2000
    replicas: 2
    instance: db.m5.4xlarge
```